Workflow (#2761)
umbraco/UmbracoDocs: Umbraco-Uno/Working-with-Forms/workflow/index.md

---
versionFrom: 8.0.0
---

# What is a workflow

> A workflow, in this case, is the flow that happens from you press the submit form button on the frontend until it ends in your email (or wherever you assigned it to arrive)

Reviewer: ```suggestion
A workflow, in this case, is the flow that starts when you press the "submit form" button on the frontend, and ends with the form reply being sent to your email (or wherever you assigned it to arrive)
```

> The most common Workflow is to have the user press the submit button, then a thank you message pops up, and then the email is sent to a designated mail inbox. However, you could choose to have it sent to, for example, slack if you wanted. In this guide, you will learn to set-up a basic workflow for your forms.

Reviewer: ```suggestion
The most common Workflow is to have the user press the submit button, present a thank you message, and then send the email to a designated mail inbox. However, you could choose to have it sent to, for example, a Slack channel if you wanted. In this guide, you will learn to set-up a basic workflow for your forms.
```

:::tip

> This requires you to have set up a form if you dont have a form set-up yet you can learn how to do that in the [Setting up a form](../Setting-up-a-form/index.md) setion.

Reviewer: ```suggestion
This requires you to have set up a form. If you don't have a form set-up yet, you can learn how to do that in the [Setting up a form](../Setting-up-a-form/index.md) section.
```

:::

## How do you set up a workflow

In this part, we will learn how to enter the configuration menu for workflows.
We will be working with a form that contains the following:

* A Name field using a short answer
* An Email field using a short answer
* A Question using a long answer

This is how you get into the configuration of your workflow:

> 1. In the Forms section of the backoffice.

Reviewer: ```suggestion
1. Navigate to the Forms section of the backoffice.
```

2. Here, select the form you want to add a workflow to.
3. In the bottom of the page, you will find a section containing a checkmark saying ***on submit***
4. Click ***Configure Workflow***
5. The pop op window will contain the Workflow that currently exists
6. Select the one that says *Sending template email to*

### Filling in the Workflow

> In this section, we will make the Workflow send to your email of choice. Please note that this is for a workflow that contains a template. To set up the one without a template, it is the same except the 8th step.

Reviewer: ```suggestion
In this section, we will make the Workflow send an email to your inbox of choice. Please note that this is for a workflow that contains a template. To set up the one without a template, it is the same approach except for the 8th step.
```
